srcpkgs/asm6/files/asm6.1

```diff
@@ -0,0 +1,314 @@
+.TH asm6
```

Comment:
Where did you find the man page? Please don't vendor it in the `void-packages` tree